docs/content/users/configuration/config.md

```diff
@@ -353,6 +330,19 @@ When `true`, the `config.*.yaml` file with the option will have its settings *ov
 
 See [Extending `config.yaml` with Custom `config.*.yaml` Files](../extend/customization-extendibility.md#extending-configyaml-with-custom-configyaml-files).
 
+## `performance`
```

Comment:
`performance_strategy` seems more specific, but couldn’t it be shorter and more specific as `mount_strategy`, `sync_strategy`, `io`, or something that more appropriately describes that it’s specific to file syncing or I/O? I’m pretty sure these are lame suggestions, but I imagine there’s a way to be more specific than `performance` _and_ `performance_strategy` without going too far.

Reply:
I don't think people understand the concepts of mount or sync, too technical and obscure, but they do generally understand "performance strategy".  Open to (even more) suggestions!

Reply:
I could imagine “performance strategy” relating to network or memory—some imaginary mode that only allows the most recent project to be spun up at any given time—where “file strategy” or something more I/O-specific would hint at what kind of performance the setting affects. I’m over-explaining because I’m reaching for ideas. 😄

Reply:
I want to say “file acceleration strategy” because it works naturally with `off`, `mutagen`, and `nfs`—but that seems like a needlessly long phrase. What I like is that if I choose “off” I’m not disabling “performance” but a specific kind of ... acceleration or augmentation that’s intended to boost _file_ performance.

Reply:
"acceleration" makes sense, probably longer and more obscure. I'm in favor of "none". "mutagen" and "nfs"

Reply:
Sorry, I meant `none`—I was referencing your comments below and still managed to get it wrong.

I guess `file_acceleration` is my champion despite its length because it specifically describes the area of concern and what’s supposed to happen. Setting `file_acceleration: none` seems like a reasonable opt-out of some bonus I/O feature, where setting `performance_strategy: none` seems more like opting out of ... performance.

Reply:
What about just "acceleration"?

Reply:
It’s probably millimeters ahead of `performance_strategy` for being a smidge more specific, but `acceleration` and `performance_strategy` both fail to convey what’s being strapped to a rocket.

Reply:
I realize I can look at the available values to understand what it’s for, I’m just being picky because it seems better if the setting name gets me correctly guessing at what the values might be.

Reply:
The thing we *don't* want is people thinking they're "turning off performance". 

And of course as other docker providers gain steam and virtiofs matures, we may not need mutagen any more. Could happen. And we still won't want people to think they're "turning off performance". Thanks for thinking about this!

Reply:
Thanks for discussing this but what's the conclusion?

Reply:
@gilbertsoft implemented performance_strategy before seeing this full conversation. We'll sort out final conclusion.

Reply:
I guess we'll just do `performance_strategy` and hope it works out. `performance_mode` would have been a possible consideration.

Reply:
And then we decided to go with `performance_mode` suggested by ChatGPT. So that will be coming. Easier to spell and shorter.